packages/utils/src/validate/rules/type.ts

```
@@ -120,9 +120,9 @@ export default function (rule, value, source, errors, options) {
 
   if (custom.includes(ruleType)) {
     if (!types[ruleType](value)) {
-      errors.push(util.format(options.messages.types[ruleType], '', rule.type))
+      errors.push(util.format(options.messages.types[ruleType], rule.type))
     }
   } else if (ruleType && typeof value !== rule.type) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)